src/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedDocuments.cs

```diff
@@ -767,5 +767,123 @@ internal static class SignedDocuments
+            "33795EB2D84BFAA3F96E5930F64EC6A94C6FD36A300E0603551D0F0101FF0404" +
+            "0302078030130603551D25040C300A06082B0601050507030330770603551D1F" +
+            "0470306E3035A033A031862F687474703A2F2F63726C332E6469676963657274" +
+            "2E636F6D2F736861322D617373757265642D63732D67312E63726C3035A033A0" +
+            "31862F687474703A2F2F63726C342E64696769636572742E636F6D2F73686132" +
+            "2D617373757265642D63732D67312E63726C304C0603551D2004453043303706" +
+            "096086480186FD6C0301302A302806082B06010505070201161C68747470733A" +
+            "2F2F7777772E64696769636572742E636F6D2F4350533008060667810C010401" +
+            "30818406082B0601050507010104783076302406082B06010505073001861868" +
+            "7474703A2F2F6F6373702E64696769636572742E636F6D304E06082B06010505" +
+            "0730028642687474703A2F2F636163657274732E64696769636572742E636F6D" +
+            "2F446967694365727453484132417373757265644944436F64655369676E696E" +
+            "6743412E637274300C0603551D130101FF04023000300D06092A864886F70D01" +
+            "010B0500038201010045B9D9868E494BD02635D0E42DDE80B37A865C389CFDD9" +
+            "9BFC9B62E2C169A73B5EABF282607439EFF5C61630886DEB63415B53683446A7" +
+            "3041686C326BA35FF0029FEF603D7C80FA0177A4DE35013529B01F759FD50414" +
+            "79BDBB6B93B18144CB14E431BC144146848EF8ADB0E28952EAD1BB49E8547FFE" +
+            "9934817036338B20C4E0B9D7C6A4E5BE3D57157F21904A5C864946313EA6B7D9" +
+            "50EE0235B5D2CD01490AD2B2A1AB5F66EC8986D64A1D9D239C131E09E5CA1C02" +
+            "A75F2D7EC07E4C858856A6A58AB94DEAC8B3D3A5BBF492EE2463B156E6A0660B" +
+            "B452E35922D00456F0DEE0ED15A8BF8FFF31008756B14EEE0AC14BCF19A3CD16" +
+            "819DC990F5F45CDE21318201B2308201AE0201013081863072310B3009060355" +
+            "04061302555331153013060355040A130C446967694365727420496E63311930" +
+            "17060355040B13107777772E64696769636572742E636F6D3131302F06035504" +
+            "03132844696769436572742053484132204173737572656420494420436F6465" +
+            "205369676E696E6720434102100D85090F3FACFF0A9008A12A9FB00A54300D06" +
+            "096086480165030402010500300D06092A864886F70D01010B050004820100E2" +
+            "980C5A30EC00729D1CFA826D7A65B43FF6806B5E0ABA23A78E4F1CAA3F6436EF" +
+            "00941C6947A9B8F20D0757B5346CF640AA217F7361BEEFF2BC997FB1D3597BF3" +
+            "D7457BD4A94062FB03660F9D86710BE2FC99876A848251F4965E1B16192714C8" +
+            "F9788C09CCDE83603ADC919297BA496E921B95F3BD9554A873E09912640FCFAA" +
+            "D9DD1441D1851E637031D390C038223AE64B048E806462DDBAC98C156BE2EE47" +
+            "2B78166BDB1612848B535ADC3F0E7BE52991A17F48AFDCCC1698A236BA338930" +
+            "50EBAAC4460DAA35185C16670F597E0E6E0CB0AA83F51AAEF452F3367DD9350A" +
+            "8A49A5A8F79DF8E921303AB5D6646A482F0F59D9980310E1AE3EE8D77CB857").HexToByteArray();
+
+        internal static readonly byte[] RsaPkcs1signedSha1DeclaredSha256WithRsa = (
```

Comment:
the `S` in `signed` should be uppercase :)

Reply:
Doh. 🐪